pages/UI.php

```diff
@@ -33,19 +35,19 @@
  *
  * @return void
  */
-function DisplayWelcomePopup(WebPage $oP)
+function DisplayWelcomePopup(WebPage $oP): void
 {
-	if (!Session::IsSet('welcome'))
-	{
-		// Check, only once per session, if the popup should be displayed...
-		// If the user did not already ask for hiding it forever
-		$bPopup = appUserPreferences::GetPref('welcome_popup', true);
-		if ($bPopup)
-		{
-			TwigHelper::RenderIntoPage($oP, APPROOT.'/', 'templates/pages/backoffice/welcome_popup/welcome_popup');
-			Session::Set('welcome', 'ok');
+	if (true || !Session::IsSet("welcome")) {
```

Comment:
```suggestion
	if (!Session::IsSet("welcome")) {
```

Woops, I forgot to remove this.